template.php

```diff
@@ -133,6 +133,29 @@ function chs_theme_preprocess_islandora_solr_metadata_display(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess().
+ */
+function chs_theme_process_islandora_solr_metadata_display(&$variables) {
+  $solr_fields = $variables['solr_fields'];
+
+  if (isset($solr_fields['mods_accessCondition_useAndReproduction_xlinkhref_ms']) && isset($solr_fields['mods_accessCondition_useAndReproduction_ms'])) {
+    // Take the xlink and node values and create an anchor out of them.
+    $link = reset($solr_fields['mods_accessCondition_useAndReproduction_xlinkhref_ms']['value']);
+    $value = reset($solr_fields['mods_accessCondition_useAndReproduction_ms']['value']);
+
+    $anchor = "<a href='$link'>$value</a>";
+
+    if (!empty($anchor)) {
+      $variables['solr_fields']['mods_accessCondition_useAndReproduction_ms']['value'] = array(
```

Comment:
This straight up stomps on any and all modification to the values that were here previously as at this point the markup is rendered. For example if you were to put truncation on this field this completely stomps it away. Hyperlinking and truncation completely cannot work in Solr metadata though so maybe this is a moot point? This still needs to be communicated to the client. Similarly if this field had multiple values (it is a ms field after all) you squashed all the values.